yt_dlp/extractor/instagram.py

```diff
@@ -727,11 +738,15 @@ def _real_extract(self, url):
         highlights = traverse_obj(videos, (f'highlight:{story_id}', 'items'), (user_id, 'items'))
         info_data = []
         for highlight in highlights:
+            highlight['title'] = story_title
             highlight_data = self._extract_product(highlight)
             if highlight_data.get('formats'):
                 info_data.append({
                     'uploader': full_name,
                     'uploader_id': user_id,
                     **filter_dict(highlight_data),
                 })
+        if username != 'highlights' and story_id and not self._yes_playlist(story_title, story_id):
```

Comment:
since this is only for stories and not highlights, just `username` looks better in the output msgs IMO
```suggestion
        if username != 'highlights' and story_id and not self._yes_playlist(username, story_id):
```